test/check-application

```diff
@@ -924,19 +924,31 @@ class TestFiles(testlib.MachineCase):
         b.wait_visible("[data-item='newfile']")
         self.assertEqual(m.execute("head -n 1 /home/admin/newdir/newfile"), "test_text\n")
         b.click(".breadcrumb-button:nth-of-type(4)")
+        self.browser.wait_not_present(".pf-c-empty-state")
 
         # Copy/paste directory
-        self.browser.wait_not_present(".pf-c-empty-state")
         m.execute("runuser -u admin mkdir /home/admin/copyDir")
+        b.click("[data-item='copyDir']")
+        b.click("#dropdown-menu")
+        b.click("#copy-item")
+        b.mouse("[data-item='newdir']", "dblclick")
+        b.click("#dropdown-menu")
+        b.click("#paste-item")
+        b.wait_visible("[data-item='copyDir']")
+        b.click(".breadcrumb-button:nth-of-type(4)")
+        self.browser.wait_not_present(".pf-c-empty-state")
+
+        # File already exists error
         b.click("[data-item='newfile']")
         b.click("#dropdown-menu")
         b.click("#copy-item")
         b.mouse("[data-item='newdir']", "dblclick")
         b.click("#dropdown-menu")
         b.click("#paste-item")
         b.wait_visible("[data-item='newfile']")
-        self.assertEqual(m.execute("head -n 1 /home/admin/newdir/newfile"), "test_text\n")
```

Comment:
Would be nice to retain that assertion, to make sure that it *actually* skips the file. Of course you'd have to change the content of the existing file in between to make sure it doesn't get replaced.